### DISCUSSION TITLE
Get into NvChad with existing configuration

### DISCUSSION THREAD

**Discussion by MartinDelille** [NvChad/NvChad#2711] [Q&A]
Hi,

I discovered this project thank to this youtube video: https://www.youtube.com/watch?v=Mtgo-nP_r8Y

I wonder if there was an advice to move from an existing neovim configuration ? The documentation completely overwrite the existing configuration.

For information, my vimrc is currently versionned in my personal dotfile: https://github.com/MartinDelille/dotfiles/blob/master/vimrc

**Comment by siduck** [comment]
@MartinDelille you have to learn basic lua and read nvchad.com from start to end, then u could convert your config to nvchad based. But wait for 3-4 days,something's going to change :grin:

**Reply by MartinDelille** [reply]
Ok thank! Is their a PR I should track regarding these change?

**Reply by siduck** [reply]
@MartinDelille  not yet, check the pinned issue.
